src/operator/activation.cc

```diff
@@ -66,15 +66,17 @@ DMLC_REGISTER_PARAMETER(ActivationParam);
 
 MXNET_REGISTER_OP_PROPERTY(Activation, ActivationProp)
 .describe(R"code(Elementwise activation function.
-The activation operations are applied elementwisely to each array elements. The
-following types are supported:
+The activation operations are applied elementwisely to each array elements.
```

Comment:
Hi "elementwisely" occurred frequently throughout the old docs but it's not a word :). Let's say "element-wise" instead (the proper term and standard on numpy docs). Thanks!